views/profiles/show.ejs

```diff
@@ -200,43 +196,35 @@
     </div>
 </div>
 <script>
-  $("#sendBtn").click(function () {
-
-    var id = document.getElementById("profileId").value
-    var sendEmail = document.getElementById("sendEmail").value
-    /*
-    var d = 
-    {
-      email : sendEmail,
-      userid: id
-    };
-    if (sendEmail != null && sendEmail != "")
-    {
-      $.ajax({
-        type: "POST",
-        url: 'http://localhost:3000/api/notify',
-        data: d,
-        success: function(res) {
-            $(".result").html(res);    
-        },
-        error: function(err) {
-            $(".result").html(err); 
-        }
-      })
+    $("#sendBtn").bind('click', function () {
+      var id = document.getElementById("profileId").value
+      var sendEmail = document.getElementById("sendEmail").value
+      var d = 
+      {
+        email : sendEmail,
+        userid: id
+      };
+      if (sendEmail != null && sendEmail != "")
+      {
+        $.ajax({
+          type: "POST",
+          url: 'http://localhost:3000/api/notify',
```

Comment:
url should be `url: '/api/notify'` so that it will work in dev & prod (and http and https)